### ISSUE TITLE
Altair Youtube

### ISSUE THREAD

**Issue by eitanlees** [altair-viz/altair#667]
Do you all think it would be useful to have some short videos outlining the installation process to include in the docs? @palewire [mentioned it](https://twitter.com/palewire/status/978761909921968128) as being a good way to get people up and running. 

The [Jupyter Lab Docs](https://jupyterlab.readthedocs.io/en/latest/user/notebook.html) contain a lot of short ~10sec videos showing off variouse features. 

I know @jakevdp already has a [youtube channel](https://www.youtube.com/channel/UCscdxGKSj4hOaVFYvslW1-g) but a dedicated Altair channel could be used for installation instructions and maybe a tutorial or two. 

Just putting the idea out there, I don't have any experience with this stuff.

**Comment by jakevdp** [comment]
Good idea! I'd be happy to put something like that on my channel, but it will be a week or two before I can get to that.

I think another good option would be to set up prominent buttons for binder and colab so that users can start playing with plots in one click.

For binder, it's just about getting the environment working correctly in the altair_notebooks repository.

For Colab, I need to talk to folks about how to populate tutorial notebooks so that people can access them directly.

**Comment by eitanlees** [comment]
I recently made a few videos explaining certain aspects of Altair. 

- [What is Altair?](https://youtu.be/AAuPPorsmJc)
- [Altair's Visualization Grammar](https://youtu.be/U7w1XumKK60)
- [Altair's Data Types](https://youtu.be/LSEPyCqjoAg)

I think they would make a good addition to the documentation. Either as an embedded video or even just a link. 

What do you all think?

**Comment by jakevdp** [comment]
Sounds great!
